src/server/pfs/cmds/cmds_test.go

```diff
@@ -149,3 +168,62 @@ func TestDiffFile(t *testing.T) {
 		"repo", tu.UniqueString("TestDiffFile-repo"),
 	).Run())
 }
+
+// TestPlurals walks through the command tree for each resource and verb combination defined in PPS.
+// A template is filled in that calls the help flag and the output is compared. It seems like 'match'
+// is unable to compare the outputs correctly, but we can use diff here which returns an exit code of 0
+// if there is no difference.
+func TestPlurals(t *testing.T) {
+	pluralCheckTemplate := `
+		pachctl {{VERB}} {{RESOURCE_PLURAL}} -h > plural.txt
+		pachctl {{VERB}} {{RESOURCE}} -h > singular.txt
+		diff plural.txt singular.txt
```

Comment:
What's the assertion here? Are we expecting the diffs to be equal?

Reply:
Yes exactly, we know the command is working if the help test is displaying for it. Otherwise, we'll get an error back from pachctl. So we compare the output of the help commands to make sure that they are the same.